### ISSUE TITLE
Update static hex values to design system css variables

### ISSUE THREAD

**Issue by georgewrmarshall** [MetaMask/metamask-extension#13247]
*Will update ticket further with example of how to approach rogue hex values that do not have a matching design system css variable*

**User Story**
As a user, I want consistent colors used throughout MetaMask

[User story on Notion](https://www.notion.so/As-a-user-I-want-consistent-colors-used-throughout-MetaMask-aa31d68132034246bdbc0281d88cf619)
[Designs on Figma](https://www.figma.com/file/aWgwMrzdAuv9VuPdtst64uuw/MetaMask---Design-System?node-id=2773%3A2094)

**Steps to reproduce**

1. Go to VS Code and search with these settings
*Search*: `#(?:[0-9a-fA-F]{3}){1,2}` (make sure to click Use Regular Expression button)
*Files to include*: `.scss`
*Files to exclude*: `ui/css/design-system/colors.scss`
2. See all static hex values in styles files

**Screenshots**

<img width="348" alt="Screen Shot 2022-01-07 at 5 35 42 PM" src="https://user-images.githubusercontent.com/8112138/148621965-e871c40b-9ce0-498c-8e40-ed37b1712c14.png">

**Task**
Update all static hex values to design system css variables.

**Success criteria**
No static hex values in any `.scss` files.

**Comment by hmalik88** [comment]
I think it would also make sense to search the code base for deprecated variable names as well, maybe we can write a script that scours the codebase for the deprecated variables & hex values in `deprecated-colors.scss` and develop some report....or it can be done manually 🤷‍♂️

**Comment by georgewrmarshall** [comment]
@hmalik88 We could add removing deprecated colors to this ticket? Or in a separate issue that could be dependent on the above? In my mind there a few stages to this:
- replace static hex values in the code base with matching design system css variables
- replace rogue(don't match design system vars) hex values with closest possible design system css variables
- replace deprecated css vars with closest matching design system css variables
- remove unused deprecated css and sass variables

**Comment by hmalik88** [comment]
@georgewrmarshall @bschorchit Yeah, not sure if removing the deprecated css vars would make sense in this issue or a separate one, but they certainly seem related. I suppose that's up to your judgement if it makes sense to split the ticket. By removing the unused deprecated css and variables, do you mean to just remove the `deprecated-colors.scss` file? Otherwise, I don't see where else they would exist and be unused?

**Comment by georgewrmarshall** [comment]
@hmalik88 yeah removing that file completely from the code base

**Comment by georgewrmarshall** [comment]
@bschorchit this has been completed with the dark mode work!!
